misc/python/materialize/output_consistency/execution/sql_executor.py

```diff
@@ -81,19 +80,24 @@ def query(self, sql: str) -> Sequence[Sequence[Any]]:
 
     def _execute_with_cursor(self, sql: str) -> None:
         try:
-            self.last_started_query = sql
+            self.last_statements.append(sql)
             self.cursor.execute(sql)
-            self.last_completed_query = sql
         except (ProgrammingError, DatabaseError) as err:
             raise SqlExecutionError(self._extract_message_from_error(err))
         except ValueError as err:
             self.output_printer.print_error(f"Query with value error is: {sql}")
             raise err
         except InterfaceError:
             print("A network error occurred! Aborting!")
-            # The current or previous query might have broken the database.
-            print(f"Last started query: {self.last_started_query}")
-            print(f"Last completed query: {self.last_completed_query}")
+            # The current or one of previous queries might have broken the database.
+            last_statements_desc = self.last_statements.copy()
+            last_statements_desc.reverse()
+            statements_str = "\n".join(
+                f"* {statement}" for statement in last_statements_desc
```

Comment:
having that * in here will make copy and paste from the log more bothersome.

Reply:
Ok, removed it